ui/app/AppLayouts/Chat/panels/communities/CommunityWelcomeBannerPanel.qml

```diff
@@ -89,7 +89,8 @@ Rectangle {
         anchors.topMargin: Style.current.padding
         onClicked: {
             Global.openInviteFriendsToCommunityPopup(root.activeCommunity,
-                                                     root.communitySectionModule)
+                                                     root.communitySectionModule,
```

Comment:
Wouldn't it be easier to declare a default argument for the signal callback, eg:
```qml
signal openInviteFriendsToCommunityPopup(var community, var communitySectionModule, var cb = null)
```

Sry not tested to work, just wondering

Reply:
Scratch it, that's a syntax error :/

Reply:
Yeah I tried and searched, but there is no way. I agree that it would have been so much cleaner